StorageUtility/Makefile

```diff
@@ -0,0 +1,98 @@
+# Compiler and tool definitions shared by all build targets 
+CC       = gcc
+CXX      = g++
+CFLAGS   = -pipe -O2 -fPIC -Wall -W -D_REENTRANT 
+CXXFLAGS = -pipe -std=c++14 -O2 -fPIC -Wall -W -D_REENTRANT 
```

Comment:
Actually, don't need to set "-D_REENTRANT" flag because not build multithread in projects. reference http://stackoverflow.com/questions/6587245/how-to-compile-the-multithread-code-with-gcc?lq=1